### ISSUE TITLE
Item error in inventory "Treasury I, II, III" TI 10

### ISSUE THREAD

**Issue by woohwoow** [ValveSoftware/Dota2-Gameplay#5563]
### Item name

Immortal treasure II

### Description

The ability to sell TI 10 treasures is not available, namely the immortal treasure II, although bidding for everyone began on November 5th.
I have 32 immortal treasure II and I can't sell any of them because of the mark ~wait for the end of TI11, which is already over. The price tag on it falls every day, but I can’t sell them ....
https://steamcommunity.com/id/apathywave/[](url)

### Example Match ID and Timestamp

5.11.22

**Comment by woohwoow** [comment]
Support shrug and can't help except redirect to github

**Comment by aseaofmonsters** [comment]
Yeah steam support wasted 6 days of time just to tell me to fk off to this site
